README.md

```diff
@@ -153,53 +153,68 @@ The ProgramCall class is used to call IBM i programs and service programs.
 
 #### Example
 ```javascript
-const {
-  Connection, ProgramCall,
-} = require('itoolkit');
-
+const { Connection, ProgramCall } = require('itoolkit');
 const { parseString } = require('xml2js');
 
-const conn = new Connection({
+
+const connection = new Connection({
   transport: 'ssh',
-  transportOptions: { host: 'myhost', username: 'myuser', password: 'mypassword' }
+  transportOptions: { host: 'myhost', username: 'myuser', password: 'mypassword' },
 });
 
+const receiver = {
+  name: 'receiver',
+  type: 'ds',
+  io: 'out',
+  len: 'rec1',
+  fields: [
+    { name: 'number_sys_values_returned', type: '10i0', value: '0' },
+    { name: 'offset_to_sys value info_table', type: '10i0', value: '0' },
```

Comment:
Because of this offset business, this is probably a bad example to use. Here I believe it assumes the data will always come back at a fixed offset, but that's not how the API works. We should use a different program as an example.

Reply:
Ok. How about using [CHGLIBL](https://www.ibm.com/support/knowledgecenter/en/ssw_ibm_i_74/apis/qlichgll.htm) ?

Reply:
Not a bad choice, though it takes in a list of user libraries to set. Not sure, do we actually handle passing in an array of values?

Reply:
> do we actually handle passing in an array of values?

I do not think so.

Would passing a `ds` of ` CHAR(11)` work?

We do use this API to add a library to lib list in [iObj.addToLibraryList](https://github.com/IBM/nodejs-itoolkit/blob/master/lib/Toolkit.js#L1332)

Reply:
In that API, it only adds a single library to the library list, while the API supports setting up to 250 libraries.

Also, `addToLibraryList` is poorly named as it doesn't _add_ anything, since the QLICHGLL API _replaces_ the library list with whatever was passed. If you want to add a library you'd have to retrieve the current list, append it to the list, then pass that list back.

Reply:
Going to use [QUSROBJD](https://www.ibm.com/support/knowledgecenter/ssw_ibm_i_74/apis/qusrobjd.htm)